src/community/schemaless-features/mongodb-schemaless/src/test/resources/org/geoserver/schemalessfeatures/mongodb/test-data/stations/query/postQuery.xml

```diff
@@ -10,20 +10,20 @@
             <ogc:And>
                 <ogc:PropertyIsLessThan>
                     <ogc:PropertyName>
-                        measurements/values/value
+                        numericValue
                     </ogc:PropertyName>
                     <ogc:Literal>
-                        25
+                        52
                     </ogc:Literal>
                 </ogc:PropertyIsLessThan>
-                <ogc:PropertyIsEqualTo>
+                <ogc:PropertyIsGreaterThanOrEqualTo>
                     <ogc:PropertyName>
-                        measurements/name
+                        numericValue
                     </ogc:PropertyName>
                     <ogc:Literal>
-                        wind
+                        50
                     </ogc:Literal>
-                </ogc:PropertyIsEqualTo>
+                </ogc:PropertyIsGreaterThanOrEqualTo>
```

Comment:
Changed the request payload, because previous one was attempting to filter by values which are present in number of nested arrays (e.g. measurements: [values: [some values], values [other values, but with same names]]), and GeoServer can not handle that.